src/core/lib.lua

```diff
@@ -730,12 +730,16 @@ function random_bytes_from_math_random (count)
    return bytes
 end
 
+local lastseed=0
 function randomseed (seed)
    seed = tonumber(seed)
    if seed then
-      local msg = 'Using deterministic random numbers, SNABB_RANDOM_SEED=%d.\n'
-      io.stderr:write(msg:format(seed))
       -- When setting a seed, use deterministic random bytes.
+      if seed ~= lastseed then
+        lastseed = seed
+        local msg = 'Using deterministic random numbers, SNABB_RANDOM_SEED=%d.\n'
+        io.stderr:write(msg:format(seed))
+      end
```

Comment:
Right, and indeed it does not help. I looked again at the code and cannot find from where all the calls to `lib.randomseed` come, so I give this part up.